app_dart/test/request_handlers/github/webhook_subscription_test.dart

```diff
@@ -770,6 +770,36 @@ void main() {
       );
     });
 
+    test('Framework labels PRs, comment if no tests. Verify a trailing slash was added to the path prefix', () async {
+      const int issueNumber = 123;
+
+      tester.message = generateGithubWebhookMessage(
+        action: 'opened',
+        number: issueNumber,
+      );
+      when(pullRequestsService.listFiles(Config.flutterSlug, issueNumber)).thenAnswer(
+        (_) => Stream<PullRequestFile>.value(
+          PullRequestFile()..filename = 'testingénieux/davoir/trouvé/ce/truc',
```

Comment:
Thank god for being bilingue, parce que ça m'a pris way too long to come up avec ça.